mockserver_client/mock_requests_loader.py

```diff
@@ -172,6 +172,37 @@ def mock_single_request(
             timing=times(1),
             file_path=file_path,
         )
+    elif method == "PUT":
+        id_ = fhir_request["id"]
+        # noinspection PyPep8Naming
+        resourceType = fhir_request["resourceType"]
+        path = f"{('/' + url_prefix) if url_prefix else ''}/4_0_0/{resourceType}/{id_}"
+
+        payload = (
+            json.dumps(
+                [
+                    {
+                        "id": id_,
+                        "updated": False,
```

Comment:
minor detail but it seems like this should be "updated": True, "created": False

Reply:
I was thinking about that, but that response will be for the case only when there is an existing resource right?
IF there is no resource with that id then it will create a new resource.

same story with the `$merge` end point as well, so I thought I will just keep it as same.

I can update as suggested, either works for me, as this is just mocked response :)

Reply:
Fair enough, i guess PUT can create or modify. probably fine to leave as is.

Reply:
updated as suggested, thanks for your review